cfgtool

```diff
@@ -233,11 +227,77 @@ The user's configuration has been created under ${client_vpn_tarname} .
 Please transmit the tar over a safe channel to it's user.
 
 In order to access the VPN the user must unpack the tar, make sure the 'nobody' group and user exists on their system and execute 'openvpn openvpn-client.cfg' inside the unpacked directory as root.
+  
+Stay safe. Use crypto. Don´t drink and factorize."
+
+}
+
+__add_and_tar_user() {
+  name="$1"; shift || fail "USAGE: ${script_file} add_user <name> <output_dir>"
+  tmp_dir=$1; shift || fail "USAGE: ${script_file} add_user <name> <output_dir>"
+  client_vpn_dirname="${tmp_dir}/vpn_${VPN_NAME}_${name}"
+  client_vpn_tarname="${client_vpn_dirname}.tar"
+
+  ${script_file} add_user $name $tmp_dir
+
+  cd "`dirname "$client_vpn_dirname"`"
+  tar -cvf "$client_vpn_tarname" "`basename "$client_vpn_dirname"`"
+
+  er "
+The user's configuration has been created under ${client_vpn_tarname} .
+
+Please transmit the tar over a safe channel to it's user.
+
+In order to access the VPN the user must unpack the tar, make sure the 'nobody' group and user exists on their system and execute 'openvpn openvpn-client.cfg' inside the unpacked directory as root.
+  
+Stay safe. Use crypto. Don´t drink and factorize."
+
+}
 
+__add_and_zip_user() {
+  name=$1;
+  password=$2;
+  tmp_dir=$3;
+  client_vpn_dirname="${tmp_dir}/vpn_${VPN_NAME}_${name}"
+  client_vpn_zipname="$vpn_${VPN_NAME}_${name}.zip"
+
+  ${script_file} add_user $name $tmp_dir
+  cd $client_vpn_dirname    
+  zip -P $password -r $client_vpn_zipname ./
```

Comment:
I've just done a quick bit of research: zip encryption is rather weak, but 7z supports proper AES256 crypto. Can we switch to that?

I was mainly trying to avoid using gpg because I suppose nobody would be able to figure out how to decompress gpg encrypted files :)